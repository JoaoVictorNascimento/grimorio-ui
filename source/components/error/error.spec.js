import '../../../internals/test/helper';

import Error from './error-component';

/** @test {Error} */
describe('Error component', () => {
/** @test {Error#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Error />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
