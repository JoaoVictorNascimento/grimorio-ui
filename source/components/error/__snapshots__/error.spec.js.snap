// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component render correctly with default values 1`] = `
"<div className=\\"wrap\\">
  <p className=\\"title\\">
    Ops, algo deu errado :(
  </p>
</div>"
`;
